ejb/remote/roles-allowed/pom.xml

```diff
@@ -45,6 +45,20 @@
                 </dependency>
             </dependencies>
         </profile>
+
+        <profile>
+            <id>payara-micro-remote</id>
```

Comment:
Other cases of skipping are done using a `skipX` property, is there a reason why you haven't used this pattern here?

Reply:
created skipRemoteEJB property